supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -73,6 +73,16 @@ For example, you might combine existing product name attributes to create compou
 :name-spring-reactive: {name-runtime-spring-boot} with {name-runtime-vertx} reactive components
 ----
 
+[[non-breaking-spaces]]
+== Non-breaking spaces
+
+Use a _non-breaking space_ (`+{nbsp}+`) between the words "Red" and "Hat". The non-breaking space prevents an automatic line break from separating the two words onto two lines. 
+
+.Example non-breaking space
```

Comment:
To be consistent with our other AsciiDoc examples, should prefix it with "Example AsciiDoc:"

```suggestion
.Example AsciiDoc: Non-breaking space
```